src/pages/Home.svelte

```diff
@@ -14,3 +15,14 @@
 <NextButton>
   { $_('continue') }
 </NextButton>
+<AlphanumericInput
+  value="T890SOLA6RF6628GB8"
+  chunkLength={4}
+  onSuccess={() => alert('success')}
+/>
+<AlphanumericInput
+  value="T890SOLA6RF6628GB8"
+  chunkLength={2}
+  onSuccess={() => alert('success')}
```

Comment:
Display something in the console instead of an alert that break the flow.

Reply:
💩